CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [next rel] - TBD
+### Added
```

Comment:
Shall we add the PR number for each of these updates? Looks like we've been doing that in the past.

Reply:
yes, good point, will do !